frontend/dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine as builder
+# Set the working directory to /app inside the container
+WORKDIR /app
+# Copy app files
+COPY . .
+# Install dependencies (npm ci makes sure the exact versions in the lockfile gets installed)
+RUN npm install 
+# Build the app
+RUN npm run build
+
+# Expose port
+EXPOSE 3000
+# Start app
+CMD ["npm", "run", "dev"]
```

Comment:
Why is this service ran with dev profile in Dockerfile?

Reply:
from packet.json:
<img width="331" alt="image" src="https://github.com/Veermove/zpi/assets/73716334/cb8085f2-c55b-4d19-af42-018b4dd814eb">
Profile dev is equivalent for next dev so it's an easy way to run nextjs app.

Reply:
`npm dev` will start a dev server, which is useful for local development. While running in container, `npm run build` and `npm run start` will do the job